subtrack.MAUI/Pages/UpsertSubscription.razor

```diff
@@ -33,6 +33,21 @@
     <div class="form-group" style="padding-top: 10px;">
         <InputTextArea class="form-control" rows="3" placeholder="Description" @bind-Value="@_subscription.Description"></InputTextArea>
     </div>
+    <div style="display: flex;padding-top: 10px;">
+        <label style="white-space: nowrap; padding-top: 5px; padding-right: 10px; font-size: 17px;">Billing Cycle every </label>
+    </div>
+    <div class="d-flex flex-column flex-sm-row" style="padding-top: 10px;">
+        <div class="flex-fill me-sm-4 me-md-4 me-lg-4 mb-4">
+            <InputNumber class="form-control" placeholder="Billing Interval" @bind-Value="@_subscription.BillingInterval" id="billingInterval"></InputNumber>
+        </div>
+        <div class="flex-fill">
+            <InputSelect @bind-Value="@_subscription.BillingOccurrence" class="form-control">
```

Comment:
Think the comment got lost but dont forgett to check the acceptance criteria/mockups about occurrence pluralization

Reply:
![image](https://github.com/Code2Gether-Discord/subtrack/assets/12110372/b984405e-728f-45cb-a500-86a149747f65)

Reply:
Nice! 
Move down every one line and do the inputs as in the previously specified format

every {1} {month}

Reply:
![image](https://github.com/Code2Gether-Discord/subtrack/assets/12110372/707c86f9-225c-4b6e-85a6-646016cbaff2)

Reply:
Interval could be a lot smaller. Wouldnt expect someone to enter more than 2 numbers

Reply:
![image](https://github.com/Code2Gether-Discord/subtrack/assets/12110372/419c654a-5f40-4baa-b5e5-d3d0ee90b335)